src/main/webapp/room.js

```diff
@@ -1,4 +1,4 @@
-const createUser = (ev) => {
+var createUser = async (ev) => {
```

Comment:
any reason to change from const to var?

also, when using javascript's latest few versions (which are supported on chrome), it is better to stick to either using just const/let or  just vars, which is the historical way to declare variables in js.

Reply:
@alekseychaikovsky oh yea, was testing something, forgot to change it back